src/selectors/edges.js

```diff
@@ -96,3 +98,20 @@ export const getVisibleEdges = createSelector(
         target: edgeTargets[id] || transitiveEdges.targets[id],
       }))
 );
+
+/**
+ * Returns an node ID where source edge is a parameter
+ */
+export const getEdgeswithParams = createSelector(
```

Comment:
It is a bit weird for this selector to be added in the context of this PR as it doesn't seem to relate to the plotly feature - I also noticed that this selector is not used in any of the newly added code in this PR as well. I wonder what this selector is used for?

Reply:
yes, this is an old selector which I removed ages ago from my last ticket related to parameters. I have no clue, how it got added. It's not in the main. :S